crits/standards/parsers.py

```diff
@@ -160,45 +253,179 @@ def parse_observables(self, observables):
         :param observables: List of STIX observables.
         :type observables: List of STIX observables.
         """
+
+        analyst = self.source_instance.analyst
         for obs in observables: # for each STIX observable
             if not obs.object_ or not obs.object_.properties:
-                self.failed.append(("No valid object_properties was found!", type(obs).__name__, obs.id_)) # note for display in UI
+                self.failed.append(("No valid object_properties was found!",
+                                    type(obs).__name__,
+                                    obs.id_)) # note for display in UI
                 continue
             try: # try to create CRITs object from observable
                 item = obs.object_.properties
-                cls = self.get_crits_type(item) # determine which CRITs class matches
-                obj = cls.from_cybox(obs)
-                obj.add_source(self.source)
-                obj.save(username=self.source_instance.analyst)
-                self.imported.append((cls._meta['crits_type'], obj)) # use class to parse object
+                if isinstance(item, Address):
+                    if item.category in ('cidr', 'ipv4-addr', 'ipv4-net',
+                                         'ipv4-netmask', 'ipv6-addr',
+                                         'ipv6-net', 'ipv6-netmask'):
+                        imp_type = "IP"
+                        for value in item.address_value.values():
```

Comment:
This is causing error `Address: 'list' object is not callable`.
It looks like the parentheses need to be removed from the end:
`for value in item.address_value.values:`
Same for the domains below.